### PR TITLE
Update Compute cloud deployment docs

### PR DESCRIPTION
**Pull request by LauraLangdon**
Per Issue #166:

> * [x]  Rearrange Cloud Deployment docs in sidebar
> * [x]  Add AWS instructions

### CODE REVIEW

File: website/docs/compute/deployment/cloud-deployment/configure-storage.md
Comment (ospencer) [inline website/docs/compute/deployment/cloud-deployment/configure-storage.md:18]: We should specify that they need to be added to the yaml file, and also show them in the example. I also wonder if we want to keep this bit about the Kubernetes secret. Maybe we have it link to the other tab?

File: website/docs/compute/deployment/other-deployments/minikube.md
Comment (ospencer) [inline website/docs/compute/deployment/other-deployments/minikube.md:64]:
```suggestion
We are disabling the built-in TLS certificate provisioning, as `ngrok` already takes care of this for us.
```

File: website/docs/compute/deployment/other-deployments/minikube.md
Comment (ospencer) [inline website/docs/compute/deployment/other-deployments/minikube.md:196]: This feels a bit off of our usual tone in the docs.

File: website/docs/compute/deployment/cloud-deployment/configure-storage.md
Comment (LauraLangdon) [reply website/docs/compute/deployment/cloud-deployment/configure-storage.md:18]: We do specify the different yaml files in the next paragraph down: 

> For Kubernetes deployments, this is done in `.suborbital/scc-controlplane-deployment.yaml` under the `controlplane` and `builder` containers sections, and for local docker-compose deployments, this is done in `docker-compose.yaml` under the `scc-control-plane` and `scc-builder` services.

> and also show them in the example

Do you mean show the file name in the yaml codeblock? I considered that but decided against it since there are two candidates for the file name depending on whether a deployment is on Kubernetes or local. 

> I also wonder if we want to keep this bit about the Kubernetes secret. Maybe we have it link to the other tab?

I'll add a link to the GCS tab!

File: website/docs/compute/deployment/other-deployments/minikube.md
Comment (LauraLangdon) [reply website/docs/compute/deployment/other-deployments/minikube.md:196]: This isn't "mine", as such; it was merged in PR https://github.com/suborbital/docs/pull/158 (I think it's showing as a change here because I had a stub for it before the other was merged and had to swap out the stub for the merged one. Probably there's a better way to do that, my bad).

I do like an aggressively upbeat and fun tone, though. 😊 If you'd rather we stay more neutral than cheerleader, I can change it ofc!

**Review by flaki — CHANGES_REQUESTED** [review]
This looks great Laura! Love that we are using the tabs for other stuff than just runnables.

One thing though, I think you rebase this PR before merging, I trust that's why GitHub is complaining of conflicts.

Otherwise, and pending others' comments, LGTM.

**Comment by LauraLangdon** [comment]
> One thing though, I think you rebase this PR before merging, I trust that's why GitHub is complaining of conflicts.

I don't think so; the conflict was with [this change](https://github.com/suborbital/docs/pull/167/#discussion_r964147547).

**Review by flaki — APPROVED** [review]

**Pull request merged**
